src/main/java/net/dv8tion/jda/core/entities/impl/EmoteImpl.java

```diff
@@ -71,6 +71,7 @@ public EmoteImpl(long id,  JDA api)
         this.id = id;
         this.api = api;
         this.guild = null;
+        this.roles = null;
```

Comment:
Forgive me, Github had collapsed the code such that it looked like this was part of the the `public EmoteImpl(long, Guild)` constructor.